common.sh

```diff
@@ -90,12 +90,12 @@ function mavenMetadataPath() {
   echo $metadataPath
 }
 
+# shellcheck disable=SC2016
```

Comment:
why is that needed? should no longer be if you get rid of `'${project.version}'`